skfda/exploratory/depth/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ._depth import (band_depth,
+                     modified_band_depth,
+                     fraiman_muniz_depth,
+                     outlyingness_to_depth)
```

Comment:
``projection_depth`` was excluded from this import for any reason?

Reply:
`projection_depth` is not added directly as it is not a functional depth, but the submodule for multivariate depths and the documentation are added in 0fafea0fb1b22a6167a23f446dd9b345d16e2568.